mapping/process-level-procurement.csv

```diff
@@ -3,7 +3,7 @@ Procuring entity,Enter name of the organisation carrying out the procurement,Pro
 Procuring entity contact details,Postal and Electronic address,Project Level: Publish the postal address in ``parties.address`` and the electronic address in ``parties.contactPoint``,OCDS: Check for a party with 'procuringEntity' in ``.roles`` and copy from ``.address`` and ``.contactPoint``,
 Contract administrative entity,Enter name of the organisation carrying out the contract administrative entity if different from the Procuring Entity,Project Level: Add an entry to ``parties`` with 'administativeEntity' included in its ``.roles`` | Contracting process: Record the name and identifier in ``.summary.tender.administrativeEntity``,OCDS: Check for a party with 'administrativeEntity' in ``.roles`` and copy from ``.name`` and ``.identifier``,
 Contract status,"Select from pre-award, active or closed",Contracting process: Publish in ``.summary.status`` using the contractingProcessStatus codelist,OCDS: Map following business logic in the contractProcessStatus codelist,`Issue #1 <https://github.com/open-contracting/infrastructure/issues/1>`_
-Procurement process,"Develop a list such as International Competitive Bidding, National Competitive Bidding, Donor Procurement Rules, Framework, Direct Award",Contracting process: Publish in ``.summary.tender.procurementMethodDetails`` and map to ``.summary.tender.procurementMethod``,OCDS: Copy from ``tender.procurementMethod`` and ``tender.procurementMethodDetails``,
+Procurement process,"Develop a list such as International Competitive Bidding, National Competitive Bidding, Donor Procurement Rules, Framework, Direct Award",Contracting process: Publish in ``.summary.tender.procurementMethodDetails`` and map to ``.summary.tender.procurementMethod``,"OCDS: Copy from ``tender.procurementMethod``; where the OCDS data uses the list required for infrastructure monitoring then copy from ``tender.procurementMethodDetails``, otherwise map to this list from ``tender.procurementMethodDetails`` where possible",
```

Comment:
> if the OCDS data uses the list developed for infrastructure monitoring, copy from ``tender.procurementMethodDetails``; otherwise, map from ``tender.procurementMethodDetails`` to this list where possible

Use 'developed' instead of 'required' to match IDS language and avoid normative word. Prefer 'if' to 'where'.